### PR TITLE
Fix pebble on macos tests

### PR DESCRIPTION
**Pull request by paulgb**
Leaving this as a draft, as the issue _seems_ to have resolved itself after a restart, but keeping in case it comes up again.

### CODE REVIEW

File: plane/plane-tests/tests/common/resources/pebble.rs [hunk 1]
```diff
@@ -182,6 +186,34 @@ impl Pebble {
...
 
+/// For some reason, Docker on Mac (Sequoia) will sometimes 500 when mounting the pebble directory
+/// inside the scratch directory. For whatever reason, first starting a dummy container that mounts
+/// the scratch directory itself (i.e. the parent of the pebble directory) seems to prevent this
+/// from happening.
+#[cfg(target_os = "macos")]
+pub async fn avoid_weird_mac_bug(name: &str, scratch_dir: &Path) -> Result<()> {
+    println!(
+        "Creating dummy container for macos {}",
+        scratch_dir.to_str().unwrap()
```
Comment (michaelsilver) [inline plane/plane-tests/tests/common/resources/pebble.rs:197]: Nit: I have a slight preference for `to_string_lossy()` but it's not a big deal.

File: plane/plane-tests/tests/common/resources/pebble.rs [hunk 2]
```diff
@@ -182,6 +186,34 @@ impl Pebble {
...
+    );
+    let docker = Docker::connect_with_local_defaults()?;
+    let name = format!("dummy-{}", name);
+
+    let config = Config {
+        image: Some("alpine:latest".to_string()), // not pinning because the actual container doesn't matter
+        host_config: Some(bollard::service::HostConfig {
+            binds: Some(vec![format!("{}:/mount", scratch_dir.to_str().unwrap())]),
```
Comment (michaelsilver) [inline plane/plane-tests/tests/common/resources/pebble.rs:205]: Nit: I have a slight preference for `to_string_lossy()` but it's not a big deal.

**Review by michaelsilver — APPROVED** [review]
Left a nit, but nbd

File: plane/plane-tests/tests/common/resources/pebble.rs (hunk 1 above)
Comment (paulgb) [reply plane/plane-tests/tests/common/resources/pebble.rs:197]: I have a medium-to-strong preference _against_ `to_string_lossy()`, if there are invalid unicode chars in a path I'd rather fail quickly.

**Pull request merged**